.github/workflows/build_and_test.yml

```diff
@@ -1023,6 +1023,18 @@ jobs:
         with:
           fetch-depth: 0
 
+      # Use custom DOCKER_CONFIG directory to avoid conflicts with default settings
```

Comment:
That code is copy/pasted a lot, could be use yaml anchors to just add it in wherever needed?

Reply:
Unfortunately, GitHub Actions supports only a subset of YAML, which doesn't include anchors.
See https://github.com/actions/runner/issues/1182